packages/replay-next/components/search-files/SearchFiles.tsx

```diff
@@ -92,17 +145,69 @@ export default function SearchFiles({ limit }: { limit?: number }) {
           <Suspense>
             <InlineResultsCount streaming={streaming} />
           </Suspense>
+          <FilterButton
+            active={caseSensitive}
+            toggle={() => setCaseSensitive(!caseSensitive)}
+            icon="case-sensitive"
+            tooltip="Match Case"
+          />
+          <FilterButton
+            active={wholeWord}
+            toggle={() => setWholeWord(!wholeWord)}
+            icon="whole-word"
+            tooltip="Match Whole word"
+          />
+          <FilterButton
+            active={useRegex}
+            toggle={() => setUseRegex(!useRegex)}
+            icon="regex"
+            tooltip="Use Regular Expression"
+          />
         </div>
-
-        <div className={styles.CheckboxWrapper}>
-          <Checkbox
-            dataTestId="FileSearch-IncludeNodeModules"
-            label="Include node modules"
-            checked={includeNodeModules}
-            onChange={() => setIncludeNodeModules(!includeNodeModules)}
+        <div
+          className={styles.InputWrapper}
+          data-error={didError || undefined}
+          data-test-id="IncludeFiles-SearchInput"
+        >
+          <Icon className={styles.Icon} type="folder-open" />
+          <input
+            autoFocus
+            className={styles.Input}
+            data-test-id="FileInclude-Input"
+            onChange={onChange(setIncludedFiles)}
+            placeholder="Files to include..."
+            type="text"
+            value={includedFiles}
+          />
+          <FilterButton
+            active={openEditors}
+            toggle={() => setOpenEditors(!openEditors)}
+            icon="open-editors"
+            tooltip="Search only in Open Editors"
+          />
+        </div>
+        <div
+          className={styles.InputWrapper}
+          data-error={didError || undefined}
+          data-test-id="ExcludeFiles-SearchInput"
+        >
+          <Icon className={styles.Icon} type="folder-closed" />
+          <input
+            autoFocus
+            className={styles.Input}
+            data-test-id="FileExclude-Input"
+            onChange={onChange(setExcludedFiles)}
+            placeholder="Files to exclude..."
+            type="text"
+            value={excludedFiles}
+          />
+          <FilterButton
+            active={defaultSrcFilter}
+            toggle={() => setDefaultSrcFilter(!defaultSrcFilter)}
+            icon="settings-off"
+            tooltip="Use Default Exclude Settings"
```

Comment:
We aren't doing the same thing as VS Code, so let's make this tooltip clearer about what we _are_ doing?
```suggestion
            tooltip="Exclude NPM modules"
```